src/Microsoft.Fast.Components.FluentUI/Components/Base/InputHelpers.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System.Globalization;
+
+namespace Microsoft.Fast.Components.FluentUI;
+
+internal static class InputHelpers<TValue>
```

Comment:
Maybe a name like `InputNumberHelpers` (or `NumberFieldHelpers`) would be more precise and adapted.

Reply:
It is used from `FluentSlider` as well (also a `FluentInputBase` decendant)